sysutils/edid-decode/Portfile

```diff
@@ -0,0 +1,18 @@
+PortSystem              1.0
```

Comment:
I just realized you're missing the modeline: https://guide.macports.org/#development.creating-portfile

The first line in every Portfile should be what's called the modeline, used by editors to set consistent formatting for all Portfiles.

```suggestion
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
PortSystem              1.0
```

Reply:
Added. Thank you so much for all the guidance!